src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
specify the properties your change on hover to improve optimization like in example below
![image](https://user-images.githubusercontent.com/80700247/229346509-d4bf6858-29c0-491f-a0a6-ddee8fb4626f.png)

https://developer.mozilla.org/en-US/docs/Web/CSS/transition